src/main/resources/application.yml

```diff
@@ -207,7 +207,7 @@ webclient:
   write-timeout-seconds: 10
   hmpps-auth:
     max-retry-attempts: 2
-    connect-timeout-seconds: 3
+    connect-timeout-seconds: 20
     read-timeout-seconds: 3
```

Comment:
Hi @awhodgson256, is the read time out or the connection time out we need change. I guess the application is timing out while waiting for the response rather than connecting